webextensions/browser-compat-data.json

```diff
@@ -832,7 +835,10 @@
         "support": true
       }, 
       "Firefox": {
-        "support": "47.0"
+        "support": "47.0", 
+        "notes": [
+          "In Firefox, 'runtime.lastError' is a string with the error text."
```

Comment:
I just checked this, and in Firefox it's an [Error](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error) object.